source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -403,34 +403,39 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "custom_vm_image_name = \"FILL-THIS-IN\"\n",
```

Comment:
It's not easy to see in the diff because of the other formatting changes that this project's `pre-commit` config made, but here I changed code like this:

```python
packer_config = {
    "managed_image_name": <the name of the customized VM image>
}
```

To this

```python
custom_vm_image_name = "FILL-THIS-IN"
packer_config = {
    "managed_image_name": custom_vm_image_name
}
```

Doing that allows the rest of this cell to be parsed as valid Python code.